docs/ort_genai_npu.md

```diff
@@ -21,17 +20,13 @@ onnxruntime-genai (aka OGA) is a new framework created by Microsoft for running
     1. Access the [AMD RyzenAI EA Lounge](https://account.amd.com/en/member/ryzenai-sw-ea.html#tabs-a5e122f973-item-4757898120-tab) and  download `amd_oga_Oct4_2024.zip` from `Ryzen AI 1.3 Preview Release`.
     1. Unzip `amd_oga_Oct4_2024.zip`
 1. Setup your folder structure:
-    1. Copy all of the content inside `amd_oga` to lemonade's `REPO_ROOT\src\lemonade\tools\ort_genai\models\`
-    1. Move all dlls from `REPO_ROOT\src\lemonade\tools\ort_genai\models\libs` to `REPO_ROOT\src\lemonade\tools\ort_genai\models\`
+    1. Copy the `amd_oga` folder from the above zip file, if desired
+    1. Create the system environment variable `AMD_OGA` and set it to the path to the `amd_oga` folder
 1. Install the wheels:
-    1. `cd amd_oga\wheels`
+    1. `cd %AMD_OGA%\wheels`
     1. `pip install onnxruntime_genai-0.5.0.dev0-cp310-cp310-win_amd64.whl`
     1. `pip install onnxruntime_vitisai-1.20.0-cp310-cp310-win_amd64.whl`
     1. `pip install voe-1.2.0-cp310-cp310-win_amd64.whl`
-1. Ensure you have access to the models on Hungging Face:
-    1. Ensure you can access the models under [quark-quantized-onnx-llms-for-ryzen-ai-13-ea](https://huggingface.co/collections/amd/quark-quantized-onnx-llms-for-ryzen-ai-13-ea-66fc8e24927ec45504381902) on Hugging Face. Models are gated and you may have to request access.
-    1. Create a Hugging Face Access Token [here](https://huggingface.co/settings/tokens). Ensure you select `Read access to contents of all public gated repos you can access` if creating a finegrained token.
-    1. Set your Hugging Face token as an environment variable: `set HF_TOKEN=<your token>`
 1. Install driver
     1. Access the [AMD RyzenAI EA Lounge](https://account.amd.com/en/member/ryzenai-sw-ea.html#tabs-a5e122f973-item-4757898120-tab) and  download `Win24AIDriver.zip` from `Ryzen AI 1.3 Preview Release`.
```

Comment:
```suggestion
    1. Access the [AMD RyzenAI EA Lounge](https://account.amd.com/en/member/ryzenai-sw-ea.html#tabs-a5e122f973-item-4757898120-tab) and  download `Win24AIDriver.zip` from `Ryzen AI 1.3 EA Release`.
```

Reply:
Fixed in https://github.com/onnx/turnkeyml/pull/244/commits/f50505a733fb2305890e10df9c2dd72888b0c2c7